paper_sections/02-gams.Rmd

```diff
@@ -1,32 +1,41 @@
+---
+output:
+  pdf_document: default
+  word_document: default
+  html_document: default
+---
```

Comment:
You can disregard this section; it should be removed.  It just got added automatically after I pushed "knit" in Rstudio, and it's not needed for this, as this is just a subsection.